package.json

```diff
@@ -3,34 +3,37 @@
   "version": "0.1.0",
   "description": "The frontend-app-admin-portal is a frontend that provides branded learning experiences.",
   "repository": "https://github.com/edx/frontend-app-admin-portal",
+  "browserslist": [
+    "extends @edx/browserslist-config"
+  ],
   "scripts": {
     "build": "fedx-scripts webpack",
-    "build:gh-pages": "NODE_ENV=development BABEL_ENV=development webpack --config=config/webpack.dev.config.js",
-    "deploy:gh-pages": "npm run build:gh-pages && gh-pages -d dist",
+    "build:with-theme": "THEME=npm:@edx/brand-edx.org@latest npm run install-theme && fedx-scripts webpack",
     "lint": "fedx-scripts eslint --ext .js --ext .jsx .",
     "lint:fix": "fedx-scripts eslint --fix --ext .js --ext .jsx .",
     "precommit": "npm run lint",
     "prepublishOnly": "npm run build",
+    "install-theme": "npm install \"@edx/brand@${THEME}\" --no-save",
     "start": "fedx-scripts webpack-dev-server --progress",
+    "start:with-theme": "THEME=npm:@edx/brand-edx.org@latest npm run install-theme && fedx-scripts webpack-dev-server --progress",
     "debug-test": "node --inspect-brk node_modules/.bin/jest --runInBand --coverage",
     "test": "fedx-scripts jest --coverage --passWithNoTests --maxWorkers=50%",
     "snapshot": "fedx-scripts jest --updateSnapshot"
   },
   "license": "AGPL-3.0",
   "dependencies": {
     "@babel/plugin-transform-runtime": "7.12.1",
-    "@edx/brand": "npm:@edx/brand-edx.org@^2.0.3",
+    "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
     "@edx/frontend-enterprise-catalog-search": "0.1.10",
     "@edx/frontend-enterprise-logistration": "0.1.11",
-    "@edx/frontend-enterprise-utils": "1.1.0",
+    "@edx/frontend-enterprise-utils": "1.1.1",
     "@edx/frontend-platform": "1.11.0",
     "@edx/paragon": "^16.14.5",
     "@fortawesome/fontawesome-svg-core": "1.2.35",
     "@fortawesome/free-brands-svg-icons": "5.15.3",
     "@fortawesome/free-regular-svg-icons": "5.15.3",
     "@fortawesome/free-solid-svg-icons": "5.15.3",
     "@fortawesome/react-fontawesome": "0.1.14",
-    "@redux-beacon/segment": "1.1.0",
```

Comment:
`@redux-beacon/segment` and `redux-beacon` are no longer used. They were previously used when we had Segment events dispatched via a redux integration, but those were ripped out during the migration to frontend-platform. This PR is re-adding those events that were ripped out.